src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -492,27 +486,14 @@ GLHEResponseFactors::GLHEResponseFactors(EnergyPlusData &state, std::string cons
         tmpGvals.push_back(var.at("g_function_g_value").get<Real64>());
     }
 
-    bool errorsFound = false;
-
-    if (tmpLntts.size() == tmpLntts.size()) {
-        this->numGFuncPairs = static_cast<int>(tmpLntts.size());
-    } else {
-        errorsFound = true;
-        ShowSevereError(state, format("Errors found processing response factor input for Response Factor= {}", this->name));
-        ShowSevereError(state, "Uneven number of g-function pairs");
-    }
-
+    this->numGFuncPairs = static_cast<int>(tmpLntts.size());
     this->LNTTS.dimension(this->numGFuncPairs, 0.0);
     this->GFNC.dimension(this->numGFuncPairs, 0.0);
 
     for (size_t i = 1; i <= tmpLntts.size(); ++i) {
         this->LNTTS(i) = tmpLntts[i - 1];
         this->GFNC(i) = tmpGvals[i - 1];
     }
-
-    if (errorsFound) {
-        ShowFatalError(state, format("Errors found in processing input for {}", this->moduleName));
-    }
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:513]:(style),[knownConditionTrueFalse],Condition 'errorsFound' is always false